app/models/Campaign.js

```diff
@@ -112,18 +92,47 @@ campaignSchema.statics.lookupByIDs = function (campaignIDs) {
  * @see https://github.com/DoSomething/gambit/issues/673
  */
 campaignSchema.methods.createMobileCommonsGroups = function () {
-  const campaign = this;
-  const prefix = `env=${process.env.NODE_ENV}
-                  campaign_id=${campaign._id}
-                  run_id=${campaign.current_run}`;
-
-  // Create mobile commons group with custom name based on this campaign
-  mobilecommons.createGroup(`${prefix} status=doing`)
-  .then(doingGroup => setMobileCommonsGroup(campaign, 'doing', doingGroup))
-  .then(() => mobilecommons.createGroup(`${prefix} status=completed`))
-  .then(completedGroup => setMobileCommonsGroup(campaign, 'completed', completedGroup))
-  .then(() => campaign.save())
-  .catch(err => logger.error(err));
+  const campaignId = this._id;
+  const campaignRunId = this.current_run;
+
+  function parseApiRes(res) {
+    if (res._id) {
+      // If a group is in the response, save the data for this env.
+      const env = process.env.NODE_ENV;
+      return {
+        doing: res.mobilecommons_groups[env].doing,
```

Comment:
:thinking: that would be bad for a lot of reason! 

but I can do a safety check